lib/metadata_listener/job.rb

```diff
@@ -17,6 +17,8 @@ def perform(path:, endpoint: nil, api_token: nil, services: [])
       services.map do |key|
         available_services[key].call(path: file.body.path, endpoint: endpoint, api_token: api_token)
       end
+
+      FileUtils.rm(file.body.path) if File.exists?(file.body.path)
```

Comment:
Just use `FileUtils.rm_f` and no `if` is needed.

Reply:
😞